compose/ui/ui/src/skikoMain/kotlin/androidx/compose/ui/node/RootNodeOwner.skiko.kt

```diff
@@ -263,13 +263,17 @@ internal class RootNodeOwner(
     }
 
     /**
-     * If pointerPosition is inside UIKitView, then Compose skip touches. And touches goes to UIKit.
+     * Perform hit test and return the [InteropView] associated with the resulting node
+     * in case it contains an [InteropViewCatchPointerModifier], otherwise null.
      */
-    fun hitTestInteropView(position: Offset): Boolean {
+    fun hitTestInteropView(position: Offset): InteropView? {
         val result = HitTestResult()
         owner.root.hitTest(position, result, true)
-        val last = result.lastOrNull()
-        return (last as? BackwardsCompatNode)?.element is InteropViewCatchPointerModifier
+        // TODO: check why we use BackwardsCompatNode here instead of new Element->Node based API
```

Comment:
Since this PR changes it - let's just fix that

Reply:
Done